tfsec.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: tfsec
+  version: 1.28.5
+  epoch: 2
+  description: Security scanner for your Terraform code
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - git
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 2a2a67987b477eb4a76ff42858973096138e9943
+      repository: https://github.com/aquasecurity/tfsec
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: google.golang.org/protobuf@v1.33.0
+
+  - runs: |
+      BINARY=tfsec
+      CHECK_GEN_BINARY=tfsec-checkgen
+      TAG=v${{package.version}}
+      GO111MODULE=on
+      export CGO_ENABLED=0
+      mkdir -p bin/linux
+      GOOS=linux GOARCH=amd64 go build \
+        -o bin/linux/${BINARY}-linux-amd64 \
+        -ldflags "-X github.com/aquasecurity/tfsec/version.Version=${TAG} \
+        -s -w -extldflags '-fno-PIC -static'" ./cmd/tfsec/
+      GOOS=linux GOARCH=amd64 go build \
+        -o bin/linux/${CHECK_GEN_BINARY}-linux-amd64 \
+        -ldflags "-X github.com/aquasecurity/tfsec/version.Version=${TAG} \
+        -s -w -extldflags '-fno-PIC -static'" ./cmd/tfsec-checkgen/
+      mkdir -p ${{targets.destdir}}/usr/bin
+      chmod +x bin/linux/*
+      mv bin/linux/${BINARY}-linux-amd64 ${{targets.destdir}}/usr/bin/${BINARY}
+      mv bin/linux/${CHECK_GEN_BINARY}-linux-amd64 ${{targets.destdir}}/usr/bin/${CHECK_GEN_BINARY}
```

Comment:
dont you want to rename this to tfsec?

also, this script only compile for amd64 and we have amd64 & arm64 here.

Reply:
i had mv command to rename binary package to tfsec in /usr/bin path
updated multi arch in new commit